xcodeproject/xcworkspace/schemes.go

```diff
@@ -44,30 +44,30 @@ func (w Workspace) Schemes() (map[string][]xcscheme.Scheme, error) {
 	// project schemes
 	projectLocations, err := w.ProjectFileLocations()
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to get project locations from workspace: %w", err)
 	}
 
 	isAutocreateSchemesEnabled, err := w.isAutocreateSchemesEnabled()
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to read the workspace autocreate scheme option: %w", err)
 	}
 
 	for _, projectLocation := range projectLocations {
 		if exist, err := pathutil.IsPathExists(projectLocation); err != nil {
-			return nil, fmt.Errorf("failed to check if project exist at: %s, error: %s", projectLocation, err)
+			return nil, fmt.Errorf("failed to check if project exist at: %s, error: %w", projectLocation, err)
```

Comment:
NOTE: I suggest droping the `, error: ` suffix. Somthing like this would be more consistent with the other errors: `failed to check if project exist at %s: %w`